src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -307,6 +307,10 @@ public void search(boolean onlyDataRelatedToMe) {
         String allTypesFilterQuery = SearchFields.TYPE + ":(dataverses OR datasets OR files)";
         filterQueriesFinalAllTypes.add(allTypesFilterQuery);
 
+        if (page == 0) {
```

Comment:
@pdurbin the log message makes me think we don't know why this is here? Can we find out if this is needed and why? I'd rather be sure of why we're adding in logic than just logging a  ¯\_(ツ)_/¯

Reply:
@scolapasta yes, the message could be better. It is needed (or something like it) to avoid a 500 error like this:

<img width="1263" alt="58897303-ffcfd280-86c5-11e9-8e6e-4c5cd174ccb6" src="https://user-images.githubusercontent.com/21006/59209162-aa3d6f00-8b78-11e9-8d2b-524e05f9dc62.png">

Reply:
Hey guys, I just created bd25409, which is likely to eliminate the root cause.

@pdurbin I had no chance to test this yet, would you be so kind?

Reply:
@poikilotherm I just tested your fix in bd25409 but by cherry-picking it into e4946bc but I'm still getting the same 500 error and `java.lang.IllegalArgumentException: paginationStart must be 0 or greater at edu.harvard.iq.dataverse.search.SearchServiceBean.search(SearchServiceBean.java:148)
` in the logs. Here are some screenshots:

![Screen Shot 2019-06-11 at 2 13 14 PM](https://user-images.githubusercontent.com/21006/59296066-142a4700-8c53-11e9-9bcb-cbf914221914.png)

<img width="1129" alt="Screen Shot 2019-06-11 at 2 13 10 PM" src="https://user-images.githubusercontent.com/21006/59296065-1391b080-8c53-11e9-8056-8c680b4e42e6.png">